Mobile app: Added about me, works, and contact sections

gabrielyea opened the pull request:
## Created works and about me section in the mobile app
- added a start screen app to introduce my self
- added works card to represent my projects
- added about me card 
- added contact info card

gabrielyea committed:
Refactor elements to use grid layout

gabrielyea committed:
Add card container classes for grid layout

gabrielyea committed:
Add work card divs

gabrielyea committed:
Add style for presentation work card

gabrielyea committed:
Update assets folder

gabrielyea committed:
Add more images and styles to index.html

gabrielyea committed:
Add presentation card and works card

gabrielyea committed:
Fixed padding and margins in works card

gabrielyea committed:
Add missing images for dark logos in assets

gabrielyea committed:
Add new card about-me and contact-card

gabrielyea committed:
Add style for about-me, contact cards

gabrielyea committed:
Add footer and icons to mobile app

gabrielyea committed:
Add new screenshots

gabrielyea committed:
Add new screenshot to README

gabrielyea committed:
Delete bad screenshot

gabrielyea committed:
Add new screenshot

gabrielyea committed:
Add new screenshot to README

Davidosky007 commented on index.html:
```diff
@@ -4,49 +4,218 @@
+      </section>
+    </header>
+
+    <section class="works">
```
Use CSS Grid on the main container that contains all of the cards in this section and use Flexbox to layout an individual card as instructed in the project requirement

gabrielyea replied:
# Can you explain this better please? I am confused because:
- my cards-container class already displays cards in a grid layout (my main container its called cards-container)
- all my individual cards use flex display for their inner elements 

Thanks for the feedback!

Davidosky007 replied:
The main container for the `div` with class name "card" is `section` with class name "works" and the project requires you to use display: grid to position all the `div`s with class name "card". let me know if it's not clear thanks 👍

gabrielyea replied:
# Thanks! 
I think I understand now, only the recent projects / work cards should be inside the grid, everything else outside.
is that correct?

Davidosky007 replied:
yes, 👍 all the cards should be inside a grid.

descholar-ceo replied:
:+1: This issue has been addressed successfully :heavy_check_mark:

Davidosky007 commented on index.html:
```diff
@@ -4,49 +4,218 @@
+          <header>
+            <h3 class="works-title"> My recent works </h3>
+          </header>
+          <img src="assets/img-1.png" alt="placeholder image">
```
I suggest you make decorations background images not `img` tags as stated in the design template

gabrielyea replied:
If I change the name so it makes sense as it being part of the card would that be ok? I think that image is supposed to be a project thumbnail or project preview not a background decoration.

Thanks for the feedback!! :+1:

Davidosky007 replied:
you are welcome 👍

descholar-ceo replied:
:+1: This issue has been addressed successfully :heavy_check_mark:

Davidosky007 commented on index.html:
```diff
@@ -4,49 +4,218 @@
+            </div>
+          </div>
+
+        </div>
```
The list of projects should be in a `ul` tag to group them, and `li` per project for this is the HTML element that represents an unordered list of items

descholar-ceo replied:
:+1: This issue has been addressed successfully :heavy_check_mark:

Davidosky007 commented on index.html:
```diff
@@ -4,49 +4,218 @@
+            <div class="pl-block">
+              Ruby
+            </div>
+          </div>
```
The list of items here should be in a `ul` tag to group them, and `li` per item for this is the HTML element that represents an unordered list of items

descholar-ceo replied:
:+1: This issue has been addressed successfully :heavy_check_mark:

Davidosky007 commented on index.html:
```diff
@@ -4,49 +4,218 @@
+            <div class="pl-block dark">
+              Ruby
+            </div>
+          </div>
```
The list of items here should be in a `ul` tag to group them, and `li` per item for this is the HTML element that represents an unordered list of items

descholar-ceo replied:
:+1: This issue has been addressed successfully :heavy_check_mark:

Davidosky007 commented on index.html:
```diff
@@ -4,49 +4,218 @@
+          <header>
+            <h1>
+              About me
+            </h1>
```
This should be wrapped in `h2`, `h3`, `h4` etc tags and not `h1` this is because it's a subtitle and not the main title.

descholar-ceo replied:
:+1: This issue has been addressed successfully :heavy_check_mark:

Davidosky007 commented on index.html:
```diff
@@ -4,49 +4,218 @@
+      <div class="card">
+        <div class="card-experience">
+          <section>
+            <h4 class="dark"> <span class="figure-1"></span> Language </h4>
```
I think you should use the proper font family for this text as it is in the design template

descholar-ceo replied:
:+1: This issue has been addressed successfully! :heavy_check_mark:

Davidosky007 reviewed (changes requested):
Hi @gabrielyea, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I suggest you begin your commit messages with a capital letter as this is best practice.
* Please check comments below, make the necessary changes and submit for review using this same PR 👍 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

gabrielyea committed:
Change placeholder name to project-thumbnail

gabrielyea committed:
Change image name so it makes sence in the project context

gabrielyea committed:
Only works card are now displayed inside the cards container grid

gabrielyea committed:
Add InamiMathi to webfonts folder

gabrielyea committed:
Change the programming languages section so it displays a list

gabrielyea committed:
Fix styling in list so they display correctly

gabrielyea committed:
Add missing font InuiMathi to webfonts folder

gabrielyea commented:
# New push! fixed requested changes
- Works/projects cards are now displayed inside cards-container as lists elements  
- Made works/projects cards children of card-container so they are the only ones displayed in the grid layout
- Made programming languages sections into a list as requested
- Added missing font InaiMathi to the about me section

descholar-ceo reviewed (changes requested):
Hi <STUDENT_MENTION>, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

- [x] I suggest you begin your commit messages with a capital letter as this is best practice.
 => The commit message should be in imperative mode or present tense, and they should be capitalized, kindly watch [this video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6) it will walk you through how you can edit your commit messages, and make them follow those details

_Tip: You can read [this article](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/) To know more about professional commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gabrielyea commented:
>     * I suggest you begin your commit messages with a capital letter as this is best practice.
>       => The commit message should be in imperative mode or present tense, and they should be capitalized, kindly watch [this video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6) it will walk you through how you can edit your commit messages, and make them follow those details

Thanks for showing me that video! I had no idea that you could re write your commits so I was really confused about how to fix them!

gabrielyea commented:
# Fixed commit messages!
- Corrected commit messages by starting them with capital letter using git rebase command

henatan99 reviewed (commented):
Hi Gabrielyea,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

henatan99 reviewed (approved):
Hi Gabrielyea,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
